.markdownlint.json

```diff
@@ -3,13 +3,17 @@
   "MD002": false,
   "MD004": { "style": "dash" },
   "MD007": { "indent": 2 },
-  "MD013": { "line_length": 600 },
+  "MD013": { "line_length": 600, "code_blocks": false },
+  "MD024": {
+    "siblings_only": true
+  },
   "MD026": false,
   "MD029": { "style": "ordered" },
   "MD033": false,
   "MD034": false,
   "MD036": false,
   "MD041": false,
+  "MD053": false,
```

Comment:
This one is disabled for temporary because of https://github.com/DavidAnson/markdownlint/issues/571